.travis.yml

```diff
@@ -4,6 +4,8 @@ dart:
   - dev
 dart_task:
   - test
```

Comment:
Why not just - test: -p vm,chrome?

Reply:
dart2js still allows dart:io imports, it wouldn't catch anything.

Reply:
Oh!